### PR TITLE
Update READMEs

### PR DESCRIPTION
**Pull request by gitmp01**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,21 +20,17 @@ Projects have the flexibility to operate pNetwork v4 either as a standalone serv
...
-pNetwork v4 introduces new on-chain entities:
+This morepo holds all the on-chain code needed for the protocol to work. In particular there are three folder of relevance:
 
-- **pNetwork Adapter contract:** needed to facilitate the wrapping and unwrapping of the ERC20 token. This would be the entry point for UIs.
-- **Lockbox:** vault where the collateral is deposited/released when wrapping/unwrapping ERC20 tokens.
-- **pNetwork fee contract:** responsible for holding all the node operator fees.
-- **pTokensV2:** the wrapped representation of the ERC20 token on the underlying chain.
-- **pNetwork Authorization Module (PAM):** verifies that the operation performed on the destination chain has been authorized by pNetwork Authorization System.
+- **solidity:** keeps all the EVM contract code, relative tests and deployment scripts
+- **cpp:** keeps all the EOSIO contract code, relative tests and deployment scripts
```
Comment (envin3) [inline README.md:28]: Shouldn't this be `ANTELOPE contract code` instead of EOSIO?

File: README.md [hunk 2]
```diff
@@ -46,7 +42,4 @@ The first release of pNetwork v4 will primarily support EVM based chains, ensuri
 - BNB chain
 - Gnosis
 - Polygon
-
-## Stay tuned!
-
-This project is currently being developed and this page will be updated in the coming days, stay tuned!
+- EOSIO (Code under `BSL` licence)
```
Comment (envin3) [inline README.md:45]: ANTELOPE instead of EOSIO

**Review by envin3 — APPROVED** [review]
EOSIO is now ANTELOPE

File: README.md (hunk 1 above)
Comment (gitmp01) [reply README.md:28]: you're right, I'm still in 2013 :sweat_smile:

**Pull request merged**